libs/application/templates/parental-leave/src/forms/OtherParentApproval.ts

```diff
@@ -7,29 +7,72 @@ import {
   Form,
   FormModes,
   coreMessages,
+  buildTextField,
+  Application,
 } from '@island.is/application/core'
 
 import Logo from '../assets/Logo'
+import { YES } from '../constants'
 import { otherParentApprovalFormMessages } from '../lib/messages'
+import { getApplicationAnswers } from '../parentalLeaveUtils'
 
 export const OtherParentApproval: Form = buildForm({
   id: 'OtherParentApprovalForParentalLeave',
-  title: otherParentApprovalFormMessages.formTitle,
+  // title: otherParentApprovalFormMessages.formTitle,
+  title: 'Tilfærsla réttinda',
   logo: Logo,
   mode: FormModes.REVIEW,
   children: [
     buildSection({
       id: 'review',
-      title: otherParentApprovalFormMessages.reviewSection,
+      // title: otherParentApprovalFormMessages.reviewSection,
+      title: 'Samþykki',
       children: [
         buildMultiField({
           id: 'multi',
-          title: otherParentApprovalFormMessages.multiTitle,
+          // title: title: otherParentApprovalFormMessages.multiTitle,
+          title: 'Samþykki á tilfærslu réttinda',
           children: [
             buildDescriptionField({
               id: 'intro',
               title: '',
-              description: otherParentApprovalFormMessages.introDescription,
+              // description: otherParentApprovalFormMessages.introDescription,
+              description:
+                'Í umsókn um fæðingarorlof hefur hitt foreldrið óskað eftir tilfærslu á réttindum frá þér til sín. Óskað er eftir tilfærslu á eftirfærandi réttindum:',
+            }),
+            buildTextField({
```

Comment:
Good idea to show the fields. I did use the description to show different copy depending on rights/allowance requests, but it's nice to follow the same pattern as the employer approval with the timeline.

However, read-only data shouldn't use the text field. It should look like this: ![Screenshot 2021-06-04 at 08 31 41](https://user-images.githubusercontent.com/937328/120771940-4b185800-c50f-11eb-8eb6-f0fb037a7fc9.png)

We have a `KeyValueFormField` that should do the job, I believe it should have the same style, if not it could be updated to follow the latest changes on the application system design system

Reply:
I implemented it here for React usage https://github.com/island-is/island.is/blob/main/libs/application/ui-components/src/components/DataValue/DataValue.tsx I saw it afterward that we had the `KeyField`. Ideally it should be refactor to use the same root component and rename to be consistent